# Node name
string node_name

# Host name
string hostname

# MAC address
string mac_addr

# IPv4 address
string ipv4_addr

# IPv6 address
string ipv6_addr
